plugin/pkg/admission/exec/admission_test.go

```diff
@@ -28,6 +28,15 @@ import (
 	"k8s.io/kubernetes/pkg/genericapiserver/registry/rest"
 )
 
+func newAllowEscalatingExec() admission.Interface {
```

Comment:
generally lgtm, but I'm not familiar with why this param combination == Allow.  
Might be worthy of a comment imo.